examples/pdc_resource_tests.rb

```diff
@@ -171,3 +171,25 @@ def token
     end
   end
 end
+
+# using prod server to test PDC::V1::ReleaseRpmMapping
+pdc = server[:prod]
+
+PDC.configure do |config|
+  config.site = pdc[:site]
```

Comment:
Identical code found in pdc_resource_tests.rb:35, pdc_resource_tests.rb:179 (mass = 104)